compiler/rustc_interface/src/interface.rs

```diff
@@ -280,6 +281,11 @@ pub struct Config {
     /// Registry of diagnostics codes.
     pub registry: Registry,
 
+    /// Enabled when a feature marked as `internal` is enabled. Makes it so that
+    /// "please report a bug" is hidden, as ICEs with internal features are wontfix,
+    /// and they are usually the cause of the ICEs.
```

Comment:
This is a bit confusing... Maybe something like
```suggestion
    /// The inner atomic value is set to true when a feature marked as `internal` is
    /// enabled. Makes it so that "please report a bug" is hidden, as ICEs with
    /// internal features are wontfix, and they are usually the cause of the ICEs.
    /// None signifies that this is not tracked.
```

Reply:
Also do we even _need_ an option? Wouldn't it be simpler to always track this and potentially ignore in the end?

Reply:
What I wanted to do with the option is to make the API surface simpler and not require _every_ tool to thread this through if they don't care enough. I don't want to make the rustc_driver API more complicated than it already is :D.
But `RunCompiler::new` could just create a new "no-op" arc instead, good point.